### PR TITLE
Pojo certifications concepts

### PR DESCRIPTION
**Pull request by enyert**
- Added POJO class for Concepts operation
- Modified Test Cases
- Added a new processor method for Concepts operation

### CODE REVIEW

File: src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java [hunk 1]
```diff
@@ -154,33 +154,25 @@ public Feeds feedDetection(@Default("#[payload]") String url) {
...
-	 *            additional transaction charge
-	 * @param linkedData Set this to 0 to hide Linked Data content links in the response
-	 * @param showSourceText Check this to include the source text in the response TwitterHandle, Hashtag, and IPAddress
-	 * @param cquery A visual constraints query to apply to the web page. Required when sourceText is set to cquery
-	 * @param xpath An XPath query to apply to the web page. Required when sourceText is set to one of the XPath values
-	 * @param sourceText How to obtain the source text from the webpage
 	 * 
-	 * @return Return a list of detected high level concepts used in the text, webpage or HTML content.
+	 * @param request Request with all the options for a combine call operation.
+	 * @return Return the concepts associated with a particular source
```
Comment (GluigiPP) [inline src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java]: Why you changed the return description? all the @return are updated

File: src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/ConceptsTestCases.java [hunk 2]
```diff
@@ -27,8 +28,9 @@ public void testWithURL() {
 	}
 
 	private void testGetConcepts(String source, String... expectedConcepts) {
-		Concepts concepts = getConnector().concepts(source,
-				null, null, null, null, null, null, null);
+		ConceptsRequest cr = new ConceptsRequest();
```
Comment (GluigiPP) [inline src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/ConceptsTestCases.java]: Please use a method like in CombineCall to generate the request, for example buildBasicRequest().
The results could look like:
`Concepts concepts = getConnector().concepts(buildBasicRequest());`

File: src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java [hunk 3]
```diff
@@ -154,33 +154,25 @@ public Feeds feedDetection(@Default("#[payload]") String url) {
...
-	 * @param knowledgeGraph Set this to 1 to include knowledge graph information in the results. This incurs an
-	 *            additional transaction charge
-	 * @param linkedData Set this to 0 to hide Linked Data content links in the response
-	 * @param showSourceText Check this to include the source text in the response TwitterHandle, Hashtag, and IPAddress
-	 * @param cquery A visual constraints query to apply to the web page. Required when sourceText is set to cquery
-	 * @param xpath An XPath query to apply to the web page. Required when sourceText is set to one of the XPath values
-	 * @param sourceText How to obtain the source text from the webpage
 	 * 
-	 * @return Return a list of detected high level concepts used in the text, webpage or HTML content.
+	 * @param request Request with all the options for a combine call operation.
```
Comment (GluigiPP) [inline src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java]: You need to be careful when you copy and paste, the proper description is "Request with all the options for the extract concepts operation."

File: src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java [hunk 4]
```diff
@@ -162,33 +159,25 @@ public Feeds feedDetection(@Default("#[payload]") String url) {
...
-	 *            additional transaction charge
-	 * @param linkedData Set this to 0 to hide Linked Data content links in the response
-	 * @param showSourceText Check this to include the source text in the response TwitterHandle, Hashtag, and IPAddress
-	 * @param cquery A visual constraints query to apply to the web page. Required when sourceText is set to cquery
-	 * @param xpath An XPath query to apply to the web page. Required when sourceText is set to one of the XPath values
-	 * @param sourceText How to obtain the source text from the webpage
 	 * 
-	 * @return Return a list of detected high level concepts used in the text, webpage or HTML content.
+	 * @param request Request with all the options for a concept operation.
+	 * @return Return the concepts associated with a particular source
```
Comment (GluigiPP) [inline src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java]: Again, don't change the return maintain the same.

File: src/main/java/org/mule/modules/watsonalchemylanguage/model/ConceptsRequest.java [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+ * 
+ * @author Admios
+ */
+public class ConceptsRequest {
+	
+	/**
+	 * Text or url to process
+	 */
+	@Default("[payload]")
+	String source;
```
Comment (GluigiPP) [inline src/main/java/org/mule/modules/watsonalchemylanguage/model/ConceptsRequest.java]: Make private all the variables

**Comment by AristidesMelendez** [comment]
This features are already implemented in another PR.

**Pull request closed without merging**